docs/faq.rst

```diff
@@ -20,3 +20,19 @@ A: No. ``@most/core`` is focused on reactive event programming rather than Array
 **Q: I want to stream chunked data with flow control (also known as "back pressure"). Should I use ``@most/core`` for that?**
 
 A: No. ``@most/core`` is focused on reactive event programming rather than chunked/block data streaming :ref:`Read more here <when should you use it>`.
+
+**Q: Are ``@most/core`` event streams the same as Node streams?**
+
+A: No. ``@most/core`` and Node streams are fairly different things, each of which happened to choose the name "stream" for one of its core concepts. They have a different primary goals, and thus different architectures and APIs.
```

Comment:
```
A: No. ``@most/core`` and Node streams are somewhat different things, each of which happened to choose the name "stream" for one of its core concepts. They have different primary goals, and thus different architectures and APIs.
```

Applying this swaps the overused word _fairly_ for the more specific word _somewhat_. Also, removing an erroneous _a_.

Reply:
What's the basis for saying "fairly" is overused?

In any case, reading this again, I think I'm in favor of refactoring the whole paragraph.  What do you think of:

> A: No. ``@most/core`` and Node streams have different primary goals, and thus different architectures and APIs.  Each happened to choose the name "stream" for one of its core concepts.

Reply:
Or this:

> A: No. ``@most/core`` and Node streams are both based on the general concept of [stream processing](https://en.wikipedia.org/wiki/Stream_processing).  However, they have different goals, architectures, and APIs.

Reply:
Just pushed an update in e1ff33b.  LMK what you think.

Reply:
Well, it is _often_ overused, and thus the meaning becomes vague instead of precise.

Vague: Dora’s new idea was fairly original.
Precise: Dora’s new idea was somewhat original.

_Fairly_ also has several meanings, e.g., _in a fair manner_, _justly_ or _honestly_, _impartially_, or _moderately_, _tolerably_, or _properly_, _legitimately_.

_Somewhat_, on the other hand, is precise as it means _in some measure or degree_, _to some extent_.

I like this phrasing:

> A: No. @most/core and Node streams are both based on the general concept of stream processing. However, they have different goals, architectures, and APIs.